src/scss/layout/_footer.scss

```diff
@@ -1 +1,73 @@
 @use 'abstracts' as *;
+
+.site-footer {
+  @extend %body__text-s;
+  position: relative;
+  display: flex;
+  align-items: center;
+
+  @include breakpoint('medium') {
+    height: rem(183px);
+  }
+
+  &::before {
+    content: '';
+    position: absolute;
+    left: calc((100vw - 100%) / -2);
+    z-index: -1;
+    display: block;
+    width: 100vw;
+    height: 100%;
+    background-color: $color__gray-500;
+  }
+
+  a {
+    color: $color__blue-200;
+
+    &:hover {
+      color: $color__blue-700;
+      text-decoration: none;
+    }
+  }
+
+  &__about {
+    display: grid;
+    grid-column-gap: rem(96px);
+    grid-row-gap: rem(16px);
+    padding: rem(48px) 0;
+  
+    @include breakpoint('medium') {
+      grid-template-columns: rem(310px) rem(400px) rem(310px); 
+    }
+  }
+
+  &__logo {
+    margin-bottom: rem(15px);
+  }
+
+  &__copyright {
+    color: $color__white-160;
+    opacity: 0.77;
+  }
+
+  &__privacy-pol {
+    @include breakpoint('medium') {
+      display: block;
+    }
+  }
+
+  &__desc,
+  &__address {
+    color: $color__white-175;
+  }
+
+  &__desc p {
+    margin-bottom: 0;
+  }
+
+  &__address {
+    width: rem(197px);
```

Comment:
Instead of setting a width, I usually put a `<br />` tag in the address to split the two lines. This ensures that even if I have to change the font size later, the lines will still split where I want them too.